From 00925ca2f05ace0fe6deb07916494e4ab305a93d Mon Sep 17 00:00:00 2001
From: Jussi Laako <jussi@sonarnerd.net>
Date: Wed, 8 Apr 2020 01:28:53 +0300
Subject: [PATCH 5/6] ALSA: usb-audio: Add Wavelength Audio to DSD detection
 whitelist

Wavelength Audio UAC IP seem to support auto-detection of DSD support,
whitelist the VID to enable DSD support. Used by number of device
vendors.

Signed-off-by: Jussi Laako <jussi@sonarnerd.net>
---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 36175473bd1d..715e49c3f6b8 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1818,6 +1818,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 	switch (USB_ID_VENDOR(chip->usb_id)) {
 	case 0x152a:  /* Thesycon devices */
 	case 0x20b1:  /* XMOS based devices */
+	case 0x21b4:  /* Wavelength Audio */
 	case 0x22d9:  /* Oppo */
 	case 0x23ba:  /* Playback Designs */
 	case 0x25ce:  /* Mytek devices */
-- 
2.25.1

